requirements.txt

```diff
@@ -33,6 +33,8 @@ django-static-jquery==2.1.4
 Django==3.2.12
 djangorestframework-simplejwt
 djangorestframework==3.13.1
+drf-spectacular-sidecar==2024.11.1
```

Comment:
It is unclear why we need these dependencies. Changeset description does not shed any light on it.

Reply:
Added description

Reply:
Changed summary to contain goal of added deps